.github/workflows/sanitizers.yaml

```
@@ -95,6 +96,18 @@ jobs:
       -
         name: Add PHP to the PATH
         run: echo "$(pwd)/php/target/bin" >> "$GITHUB_PATH"
+      - uses: actions/checkout@v4
+        name: Checkout watcher
+        with:
+          repository: e-dant/watcher
+          ref: ${{ env.EDANT_WATCHER_VERSION }}
+          path: 'edant/watcher'
+      - name: Compile edant/watcher
+        run: |
+          cd edant/watcher/watcher-c/
+          gcc -o libwatcher.so ./src/watcher-c.cpp -I ./include -I ../include -std=c++17 -O3 -Wall -Wextra -fPIC -shared
```

```suggestion
          clang -o libwatcher.so ./src/watcher-c.cpp -I ./include -I ../include -std=c++17 -Wall -Wextra -shared ${{ matrix.sanitizer == 'msan' && '-fsanitize=memory -fno-omit-frame-pointer -fno-optimize-sibling-calls' || '' }}
```

Something like that should help having MSAN support.